app/controllers/api/v1/subject_sets_controller.rb

```diff
@@ -21,13 +21,20 @@ def update_links
 
   def destroy
     resource_class.transaction(requires_new: true) do
-      controlled_resources.each do |resource|
```

Comment:
i looked up the spec and it looked like the deletes were via the resource URL so we don't need to support it here (and i don't think our rails routes ever did either)
> An individual resource can be deleted by making a DELETE request to the resource’s URL: